src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1655,7 +1655,34 @@ public void cannotCreateRequestWhenRequesterHasActiveRequestManualBlocks() {
     assertThat(postResponse.getJson(), hasErrors(1));
     assertThat(postResponse.getJson(), hasErrorWith(allOf(
       hasMessage("Patron blocked from requesting"))));
-      hasParameter("reason", "Display description");
+    hasParameter("reason", "Display description");
+  }
+
+  @Test
+  public void cannotCreateRequestWhenExpirationDateIsNotSet() {
```

Comment:
Does this test demonstrate that any request cannot be created without an expiration date or only requests for a patron that has been manually blocked from creating requests?